tendrl/gluster_integration/manager/__init__.py

```diff
@@ -78,6 +80,39 @@ def shutdown(signum, frame):
             {"message": "Signal handler: stopping"}
         )
         complete.set()
+        # Remove the node's name from gluster server tag
+        try:
+            gl_srvr_list = NS._int.client.read(
+                "/indexes/tags/gluster/server"
+            ).value
+            gl_srvr_list = json.loads(gl_srvr_list)
+            if NS.node_context.node_id in gl_srvr_list:
+                gl_srvr_list.remove(NS.node_context.node_id)
+            NS._int.client.write(
+                "/indexes/tags/gluster/server",
+                gl_srvr_list
+            )
+            NS._int.client.delete(
```

Comment:
check if this service actually is the provisioner and then delete the tag

Reply:
ok